Documentation/mgcb.md

```diff
@@ -50,13 +50,16 @@ Set the target platform for this build. It must be a member of the TargetPlatfor
 * Xbox360
 * iOS
 * Android
-* Linux
+* DesktopGL
 * MacOSX
 * WindowsStoreApp
 * NativeClient
+* PlayStationMobile
```

Comment:
Can you actually remove `PlayStationMobile` from here... it should no longer be used and has been marked obsolete in the code base.